src/main/resources/config/application-dev.yml

```diff
@@ -14,77 +14,79 @@
 # ===================================================================
 
 logging:
-    level:
-        ROOT: DEBUG
-        io.github.jhipster: DEBUG
-        com.icthh.xm.tmf.ms.qualification: DEBUG
+  level:
+    ROOT: DEBUG
+    io.github.jhipster: DEBUG
+    com.icthh.xm.tmf.ms.qualification: DEBUG
 
 spring:
-    profiles:
```

Comment:
@yevheniimuzyka 
why TAB width was changed from 4 spaces to 2? it generates a lot of unnecessary changes.